### ISSUE TITLE
SourceMap generates wrong paths

### ISSUE THREAD

**Issue by hashworks** [sass/node-sass#1194]
Example setup with grunt-sass: https://moonbase.hashworks.net/grunt-sass-issue-241.tar
```
Gruntfile.js
index.html
package.json
css
├── screen.css
└── screen.css.map
sass
└── screen.scss
```

screen.css.map:
```json
{
        "version": 3,
        "file": "screen.css",
        "sources": [
                "../screen.scss"
        ],
        "sourcesContent": [],
        "mappings": "AAAA;EAIC,AAAkB;EAHnB,AAAM;IACJ,AAAkB",
        "names": []
}
```

Should be `../sass/screen.scss`.

**Comment by saper** [comment]
Got

Whoops! Error 404

The page you're trying to access doesn't seem to exist anymore.

Check out what's available!

**Comment by xzyfer** [comment]
Thanks for the report @hashworks. Could you please created a small GitHub repo with a minimal work example reproducing this issue. This will ensure there are no other external factors. 
Also our experience the process of creating these test repos in itself surfaces the problem.

**Comment by hashworks** [comment]
Hi, sry for the dead link.

I reproduced my case with grunt-sass v0.18.1, however it seems to be fixed with grunt-sass v1.1.0-beta (which includes node-sass v3.0 and libsass v3.2.0).
